src/app/rpmostree-builtin-upgrade.cxx

```diff
@@ -58,9 +60,44 @@ static GOptionEntry option_entries[] = {
   { "unchanged-exit-77", 0, 0, G_OPTION_ARG_NONE, &opt_unchanged_exit_77, "If no new deployment made, exit 77", NULL },
   { "trigger-automatic-update-policy", 0, G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_NONE, &opt_automatic, "For automated use only; triggered by automatic timer", NULL },
   { "lock-finalization", 0, G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_NONE, &opt_lock_finalization, "Prevent automatic deployment finalization on shutdown", NULL },
+  { "force", 'f', 0, G_OPTION_ARG_NONE, &opt_force, "Force an upgrade even if an updates driver is registered.", NULL},
```

Comment:
`--force` is very generic. If we decide to have a an override CLI flag, I'd prefer something specific to the action (e.g. `--bypass-driver`).